api/cds.proto

```diff
@@ -31,86 +33,108 @@ service ClusterDiscoveryService {
   }
 }
 
-// An extensible structure containing the address Envoy should bind to when establishing upstream
-// connections.
+// An extensible structure containing the address Envoy should bind to when
+// establishing upstream connections.
 message UpstreamBindConfig {
   // The address Envoy should bind to when establishing upstream connections.
   Address source_address = 1;
 }
 
-// Circuit breaking settings can be specified individually for each defined
-// priority.
+// :ref:`Circuit breaking<arch_overview_circuit_break>` settings can be
+// specified individually for each defined priority.
 message CircuitBreakers {
+
+  // A Thresholds defines CircuitBreaker settings for a RoutingPriority.
   message Thresholds {
+    // The RoutingPriority the specified CircuitBreaker settings apply to.
     RoutingPriority priority = 1;
+
     // The maximum number of connections that Envoy will make to the upstream
-    // cluster. If not specified, the default is 1024. See the circuit
-    // breaking overview for more information.
+    // cluster. If not specified, the default is 1024.
     google.protobuf.UInt32Value max_connections = 2;
+
     // The maximum number of pending requests that Envoy will allow to the
-    // upstream cluster. If not specified, the default is 1024. See the circuit
-    // breaking overview for more information.
+    // upstream cluster. If not specified, the default is 1024.
     google.protobuf.UInt32Value max_pending_requests = 3;
+
     // The maximum number of parallel requests that Envoy will make to the
-    // upstream cluster. If not specified, the default is 1024. See the circuit
-    // breaking overview for more information.
+    // upstream cluster. If not specified, the default is 1024.
     google.protobuf.UInt32Value max_requests = 4;
+
     // The maximum number of parallel retries that Envoy will allow to the
-    // upstream cluster. If not specified, the default is 3. See the circuit
-    // breaking overview for more information.
+    // upstream cluster. If not specified, the default is 3.
     google.protobuf.UInt32Value max_retries = 5;
   }
+
+  // If multiple :ref:`Thresholds<envoy_api_msg_CircuitBreakers.Thresholds>` are defined with
+  // the same RoutingPriority, the
```

Comment:
Suggest adding links to RoutingPriority definition from the two uses in this comment block.